R-Ladies Conference Participation (#72)
rladies/rladiesguide: content/comm/conferences/index.en.md

---
title: "R-Ladies Conference Participation"
menuTitle: "Conference Participation"
weight: 110
---

R-Ladies participates in R-Ladies branded conference sessions and events such as
contributed talks, panel discussions, networking events, poster presentations, and booths to 
increase awareness of our organization.

## R-Ladies branded conference event

> An R-Ladies branded event is one that used the name `R-Ladies` to....

Reviewer: ```suggestion
An R-Ladies branded event is one that used the name `R-Ladies` in its name.
```

Is this what you mean?

Author: Yes - I did not know how to define the scope!

Author: @maelle it looks like I overwrote your suggested change - sorry, that was not my intention! Modified slightly to use the word `name` less.

R-Ladies directory members may lead R-Ladies branded conference events with the 
approval of the R-Ladies global team.

## Organizing an R-Ladies branded conference event

If you are an R-Ladies directory member interested in leading an R-Ladies branded 
event, please email the global team at `info@rladies.org` with the template below and request approval for the event.
Once approved, we will add the event to our records and offer additional support if needed.

If you need assistance identifying a candidate R-Ladies directory member to lead a session,

> please email us the template below. 

Reviewer: add direct links to the templates?

Author: good idea

Speakers invited to represent R-Ladies at an R-Ladies branded event should
have all event fees waived or reimbursed. If travel is involved, the 
event should further financially support the speaker by covering costs associated
with travel, lodging, or meals.

## Email templates

### Seeking approval for an R-Ladies branded conference event

To: 

```
info@rladies.org
```

Subject:

```
Seeking approval for an R-Ladies branded conference event
```

Body:

```
Dear R-Ladies,

The <conference/event> will be held on <dates> at <location>. <Name(s)> is seeking
approval to represent R-Ladies by <hosting/presenting> an R-Ladies branded conference 
event in the form of <contributed talk/panel discussion/networking event/booth>.

> <Abstract/More information...>

Reviewer: ```suggestion
<Abstract/More information/Link to event page...>
```

```

### Seeking an R-Ladies invited speaker for an R-Ladies branded conference event

To: 

```
info@rladies.org
```

Subject:

```
Seeking an R-Ladies invited speaker for an R-Ladies branded conference event
```

Body:

```
Dear R-Ladies,

> The <conference/event> will be held on <dates> at <location>. We would appreciate 

Reviewer: ```suggestion
The <conference/event (with link to the conference page)> will be held on <dates> at <location>. We would appreciate 
```

---
title: "R-Ladies Conference Participation"
menuTitle: "Conference Participation"
weight: 110
---

> R-Ladies participates in conference sessions such as contributed talks,

Reviewer 2: Are there any criteria for the conferences? Does it have to be conferences around only R/Statistic?

Author: Good question. I'm not sure. I can imagine there might be some conferences outside the scope of R and statistics which may interested in hearing from the org.

panel discussions, networking events, poster presentations, and booths to 
increase awareness of our organization.

## Organizing a conference event

> If you have an R-Ladies member interested in leading a conference

Reviewer: ```suggestion
If you are an R-Ladies member interested in leading a conference
```

So one only needs to be a member but what does this mean exactly, e.g. "listed in the directory", or organizer/global team member?

> session, please notify us at `info@rladies.org` for our records and so that

Reviewer 2: ```suggestion
session, please notify us at `info@rladies.org` for our records so that
```

> session, please notify us at `info@rladies.org` for our records and so that

Reviewer 2: If there's support (e.g. paying registration fees) for R-Ladies members who are interested in leading a conference, it might be useful to add it here. If not, ignore this!

Author: I was thinking this, too!

Author: Still getting the hang of review mode! I added a [commit ](https://github.com/rladies/rladiesguide/pull/72/commits/754026ce3c51887d1c3f1bb53eea5184aa434526) w/ 

> We can financially support the speaker's conference attendance by... <please 
describe>.

we may offer additional support if needed.

If you need assistance identifying a candidate R-Ladies member to lead a session,

> please email us with the template below. 

Reviewer 2: ```suggestion
please email us the template below. 
```

> It would be appreciated if fees associated with the conference could be waived

Reviewer: Should this be re-inforced somehow by explicitly stating the organization and the invited speaker will not pay for the session?
And that further financial support might help the invited speaker?

Reviewer 2: I second that - adding more details here will be useful!

Author: How about something like: 

>Speakers invited to represent R-Ladies should have all conference fees waived or reimbursed. If travel is involved, the conference should further support the speaker financially by reimbursing travel, lodging, and meals.
